web/packages/teleterm/src/services/tshd/types.ts

```diff
@@ -333,5 +333,7 @@ export type CreateConnectMyComputerRoleResponse =
 export type CreateConnectMyComputerNodeTokenResponse =
   apiService.CreateConnectMyComputerNodeTokenResponse.AsObject;
 
+export const UserType = apiCluster.LoggedInUser.UserType;
```

Comment:
We can export value and type at the same time (we may need the type in the future).
```suggestion
export { UserType }
``` 
I did it for `export { FileTransferDirection }` above.

Reply:
I wanted to do this but I didn't know how to reexport something that's defined in a namespace. I think I found a solution, let me know what you think.

Reply:
Ah, I didn't notice that it is defined in a namespace. Your solution is fine, I was afraid that clicking on `UserType` somewhere in the code will open the `types.ts` file, not the actual definition. But it work as it should!